packages/common/src/api/tan-query/useUpdateUser.ts

```diff
@@ -1,51 +1,42 @@
 import { User } from '@audius/sdk'
 import { useMutation, useQueryClient } from '@tanstack/react-query'
 
+import { userMetadataToSdk } from '~/adapters/user'
 import { useAppContext } from '~/context/appContext'
+import { ID } from '~/models/Identifiers'
+import { UserMetadata } from '~/models/User'
+import { encodeHashId } from '~/utils/hashIds'
 
 import { QUERY_KEYS } from './queryKeys'
 
-type UpdateUserMetadata = {
-  name?: string
-  bio?: string
-  location?: string
-  artistPickTrackId?: string
-  isDeactivated?: boolean
-  twitterHandle?: string
-  instagramHandle?: string
-  tiktokHandle?: string
-  website?: string
+type MutationContext = {
+  previousUser: any
```

Comment:
same here. is this not a User or UserMetadata type?

Reply:
oh yeah great find, re-adding